app/test_split_integer.py

```diff
@@ -2,20 +2,23 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(20, 3, )) == 20
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(10, 2) == [5, 5]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(10, 1) == [10]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert (
+        split_integer(32, 6)
+        == sorted(split_integer(32, 6))
+    )
```

Comment:
Do not overcomplicate the check, evaluate against actual list as it is done in the `test_should_split_into_equal_parts_when_value_divisible_by_parts`. The same relates to the `test_should_add_zeros_when_value_is_less_than_number_of_parts` test as well